tests/test_cli.py

```diff
@@ -1,5 +1,5 @@
 """Test the CLI."""
-
+import logging
```

Comment:
```suggestion

import logging
```

just a small style thing, not covered with our linter setup yet 😞

Reply:
This has been correccted